internal/driver/glfw/window.go

```diff
@@ -572,11 +578,12 @@ func (w *window) mouseClickedHandleTapDoubleTap(co fyne.CanvasObject, ev *fyne.P
 
 func (w *window) waitForDoubleTap(co fyne.CanvasObject, ev *fyne.PointEvent) {
 	ctx, mouseCancelFunc := context.WithDeadline(context.TODO(), time.Now().Add(w.driver.DoubleTapDelay()))
-	if mouseCancelFunc != nil {
-		defer func() {
-			runOnMain(mouseCancelFunc)
-		}()
-	}
+	defer func() {
+		runOnMain(mouseCancelFunc)
+	}()
```

Comment:
Just a small suggestion while I ask another question elsewhere. Would not have bothered with this otherwise:
```suggestion
	defer runOnMain(mouseCancelFunc)
```